test/dlc_tests/ec2/test_smdebug.py

```diff
@@ -58,11 +62,15 @@ def test_smdebug_cpu(training, ec2_connection, region, cpu_only, py3_only):
         hide=True,
     )
 
-    ec2_connection.run(
+    test_output = ec2_connection.run(
         f"docker exec --user root smdebug-cpu /bin/bash -c '{test_script} {framework}'",
-        hide=True,
+        hide=True, warn=True
     )
 
+    LOGGER.info(test_output.stdout)
```

Comment:
Is it necessary to always log stdout? Can we log it in line with the assertion error?

Reply:
What is the cost of dumping stdout log and why do we not want to do it?

This might help us catch useful warning messages maybe?

Reply:
The cost is simply overcrowding logs with information from unclear origin. Currently, we run tests in parallel, so logging things to stdout, while useful, can be confusing. In a case of failure, by streaming and raising, we'd have two large log dumps (assuming the change is made to also raise the messages in the assertion error below). Check the current logs for instance -- if you think they provide value, I'm happy with keeping them.